ui/transformation.py

```diff
@@ -18,46 +18,35 @@
     from nexus_constructor.transformation_view import EditTransformation
 
 
-class Ui_Transformation(object):
-    def setupUi(self, Transformation: "EditTransformation"):
+class UiTransformation:
+    def __init__(self, Transformation: "EditTransformation"):
         Transformation.setObjectName("Transformation")
         Transformation.resize(361, 171)
-        self.frame_layout = QVBoxLayout(Transformation)
-        self.frame_layout.setContentsMargins(4, 4, 4, 4)
+
         self.main_layout = QVBoxLayout()
         self.main_layout.setSpacing(4)
 
+        self.frame_layout = QVBoxLayout(Transformation)
```

Comment:
All this gui stuff should be in the setup_ui method.

Reply:
In that case class attributes are defined outside of the constructor. I do not like that. Another way of doing this making setupUI private via `_setupUI` but you do not wanna change the name already, so that is not a choice. If you say just move them whatever, I shall do, but these type of details usually bite our ..s afterwards. :)